sdcm/db_stats.py

```diff
@@ -453,6 +453,10 @@ def get_scylla_versions(self):
         versions = {}
         try:
             node = self.db_cluster.nodes[0]
+            if node.scylla_version_detailed and 'build-id' in node.scylla_version_detailed:
+                build_id = node.scylla_version_detailed.split('with build-id ')[-1]
+            else:
+                build_id = node.get_scylla_build_id()
```

Comment:
well, it could be that, in some cases, there will already be the "with build-id " in the result, so, if it doesn't have it, then we run `get_scylla_build_id()`